plugins/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+        "target": "es2015",
+        "allowSyntheticDefaultImports": true,
+        "declaration": true,
+        "outDir": "./src/core/declarations",
+        "emitDecoratorMetadata": true,
+        "experimentalDecorators": true,
+        "rootDir": "./src/core",
+        "module": "esnext",
+        "moduleResolution": "node"
+    },
+     "include":[
+        "src/core/*.ts"
```

Comment:
Can/should we glob files to allow future module additions more easily?

e.g. `src/**/*.ts`